integrations/adding-an-integration/connecting-to-slack.md

```diff
@@ -0,0 +1,46 @@
+# Connecting to Slack
+
+You can connect aqueduct to your Slack App in order to receive automated notifications on your workflow execution status. You can simply click the slack icon on your [integrations page](../../integrations.md). Aqueduct will ask for [#Bot token](connecting-to-slack#setting-up-slack-app), [#channels](connecting-to-slack#configuring-channels), and [#workflow](connecting-to-slack#configuring-global-workflow-settings) information to connect to an email integration.
+
+## Setting up Slack App
+To connect Aqueduct to slack to recieve slack notifications, you would first need to set up your own Slack App. You can follow these steps to create one:
+* Go to [Slack Apps management page](https://api.slack.com/apps).
+* Hit **Create New App** button.
+<figure><img src="../../.gitbook/assets/connecting_slack_create_app.png" alt=""><figcaption></figcaption></figure>
+
+* Select **Create from manifest** on the dialog.
+* Select a workspace you would like to enable notification.
+* Choose **yaml** and copy over the following content to the manifest. You can change **name** and **display_name** fields. This step sets up minimum required permissions and configurations for this app. You can add additional settings once the creation is done.
+    ```yaml 
+    display_information:
+        name: my-aqueduct-bot
+    features:
+    bot_user:
+        display_name: my-aqueduct-bot
+        always_online: false
+    oauth_config:
+    scopes:
+        bot:
+        - channels:join
+        - chat:write
+        - chat:write.public
+        - channels:read
+    settings:
+    org_deploy_enabled: false
+    socket_mode_enabled: false
+    token_rotation_enabled: false
+    ```
+* Once created, you will enter the app's management page. Click **Install to Workspace** and follow instructions. This completes the Slack App setup.
+* You can also use an existing app. Our minimum permission requirements are: `channels:join`, `chat:write`, `chate:write.public`, and `channels:read`.
```

Comment:
Why would the user have an existing app for Aqueduct?

Reply:
I think we could assume the user may have an app set up for some other notification purposes. Aqueduct will work as long as the user provide us the token so that we can send notification through this existing app. The only requirements is the app's permission, which is explained here.

Reply:
Hm, that seems confusing to me — I would get Aqueduct notifications for an app I set up for another purpose?